fetch-in-react-practice/client/components/app.jsx

```diff
@@ -0,0 +1,114 @@
+import React from 'react';
+import PageTitle from './page-title';
+import TodoList from './todo-list';
+import TodoForm from './todo-form';
+
+export default class App extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {
+      todos: []
+    };
+    this.addTodo = this.addTodo.bind(this);
+    this.toggleCompleted = this.toggleCompleted.bind(this);
+  }
+
+  componentDidMount() {
+    this.getAllTodos();
+  }
+
+  getAllTodos() {
+
+    fetch('/api/todos')
+      .then(res => res.json())
+      .then(data => this.setState({ todos: data }));
+    /**
+     * Use fetch to send a GET request to `/api/todos`.
+     * Then 😉, once the response JSON is received and parsed,
+     * update state with the received todos.
+     */
+  }
+
+  addTodo(newTodo) {
+    fetch('/api/todos', {
+      method: 'POST',
+      headers: { 'Content-Type': 'application/json' },
+      body: JSON.stringify(newTodo)
+    })
+      .then(res => res.json())
+      .then(newTodo => {
+        const newTodosArray = this.state.todos.concat(newTodo);
+        this.setState({
+          todos: newTodosArray,
+          isCompleted: false
```

Comment:
You can leave this part out. Try console logging `newTodo` from line `39`. It already has `isCompleted`.